camera_toolkit/utils.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
```

Comment:
this file should be deprecated once the airo-spatial-algebra package is ready for use